src/main/java/com/godaddy/sonar/ruby/simplecovrcov/CoverageReportFileAnalyzerImpl.java

```diff
@@ -0,0 +1,170 @@
+package com.godaddy.sonar.ruby.simplecovrcov;
+
+import com.godaddy.sonar.ruby.simplecovrcov.data.CoverageReport;
+import com.godaddy.sonar.ruby.simplecovrcov.data.Mark;
+import com.godaddy.sonar.ruby.simplecovrcov.data.Reporter;
+import com.godaddy.sonar.ruby.simplecovrcov.data.ReporterItem;
+import com.google.common.collect.Maps;
+import org.sonar.api.ExtensionPoint;
+import org.sonar.api.batch.BatchSide;
+import org.sonar.api.measures.CoverageMeasuresBuilder;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.SortedMap;
+
+@BatchSide
```

Comment:
I like the thoughtfulness given to moving to 6.2+ it will help us in the future :+1: